packages/f/font-ubuntu-ttf/files/ubuntu.metainfo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>Ubuntu</id>
```

Comment:
This should match the name of the package, in this case "font-ubuntu-ttf"

Reply:
done